### PR TITLE
Homework week3

### PR DESCRIPTION
**Pull request by seenachacko**

### CODE REVIEW

File: week3/webApplicationHw/Controllers/CarController.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+using webApplicationHw.Models;
+
+namespace webApplicationHw.Controllers;
+
+
+[ApiController]
+[Route("[controller]")]
+public class CarController : ControllerBase
+{
+
```
Comment (calindurnea) [inline week3/webApplicationHw/Controllers/CarController.cs]: Try avoiding new empty lines that are not used to delimit functionality or logic, same for lines: 5,38, 35, 41,48-50

File: week3/webApplicationHw/Controllers/CarController.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    public static List<Car> Cars { get; set; } = new List<Car>(){
+        new Car(){ ID=0, Brand="Fiat", Model="500", Year=2019},
```
Comment (calindurnea) [inline week3/webApplicationHw/Controllers/CarController.cs:9]: Use the `Shift + Alt + F` shortcut to auto format the code, makes everything more easily readable

File: week3/webApplicationHw/Controllers/CarController.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    [HttpPost("AddCar")]
+    public void AddCar([FromBody] Car car)
+    {
+        Cars.Add(car);
+    }
+
+    [HttpGet("ConvertionGlloontoLitter")]
+    public double ConvertionGlloontoLitter(int liters)
```
Comment (calindurnea) [inline week3/webApplicationHw/Controllers/CarController.cs:23]: You have a typo here, `Gallons`

File: week3/webApplicationHw/Controllers/CarController.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    {
+
+        return new Distance() { Miles = miles, Kilometers = miles * 1.609 };
+    }
+
+
+    [HttpPost("convert")]
+    public List<ConversionResponse> ConvertValues([FromBody] ConversionRequest request)
+    {
+        return new ConversionResponse().conversions(request);
```
Comment (calindurnea) [inline week3/webApplicationHw/Controllers/CarController.cs]: Names of methods should start with uppercase, so `Conversions` here

**Review by calindurnea — COMMENTED** [review]
Great job! 💯 
I left a few comments but nothing dramatic. As mentioned in one of them, avoid having too many empty lines.

Let me know if you have any questions 😊

File: week3/webApplicationHw/Controllers/CarController.cs (hunk 4 above)
Comment (seenachacko) [reply week3/webApplicationHw/Controllers/CarController.cs]: Thanks for the feedback

**Review by calindurnea — APPROVED** [review]

**Pull request merged**